lib/readwise/tag.rb

```diff
@@ -0,0 +1,7 @@
+module Readwise
+  Tag = Struct.new(
+    'ReadwiseTag',
+    :name,
```

Comment:
```suggestion
    :id,
    :name,
```

issue:

Can we add the ID here as an attribute as well? I don't see a reason to remove it and it would be helpful to disambiguate tags down the line.

Reply:
For sure! I assumed since we didnt have them for the book or highlight data coming in I figured we didn't want them within the tag. 😅

Reply:
We have them for the Books and Highlights ([example](https://github.com/andjosh/readwise-ruby/blob/cbb5e31f9c75079c7c76a4bbf87090a2a3ded693/lib/readwise/book.rb#L8)), but they're called `book_id` and `highlight_id` to disambiguate, so I guess we should call this one `tag_id` to keep with that pattern. I think it's a smart pattern for the future.

Reply:
Oh I see now, definite brain fart my mistake. I'll add in that change as well!